.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Such files should be not added to repository. Please remove them. Leave only index.html and js libraries that are used.

Reply:
Deleted encoding.xml file